packages/cli/src/validator/validate-locales.ts

```diff
@@ -0,0 +1,48 @@
+import chalk from 'chalk';
+import meow from 'meow';
+
+import jsonDiff from 'json-diff';
+
+import nl from '../../../app/src/locale/nl.json';
+import en from '../../../app/src/locale/en.json';
+
+function objectsHaveSameKeys(
+  obj1: Record<string, unknown>,
+  obj2: Record<string, unknown>
+) {
+  const objects = [obj1, obj2];
+  const allKeys: string[] = objects.reduce(
+    (keys: string[], object) => keys.concat(Object.keys(object)),
+    []
```

Comment:
If you type the second argument you don't have to type the function arguments. The object type is already known from objects.

```suggestion
  const allKeys: string[] = objects.reduce(
    (keys, object) => keys.concat(Object.keys(object)),
    [] as string[]
```